javascript/frameworks/cap/lib/advanced_security/javascript/frameworks/cap/CDS.qll

```diff
@@ -60,13 +60,14 @@ class CdsServeCall extends DataFlow::CallNode {
 
   Expr getServiceRepresentation() { result = serviceRepresentation }
 
-  UserDefinedApplicationService getServiceDefinition() {
+  ServiceInstance getServiceDefinition() {
```

Comment:
This should remain `UserDefinedApplicationService` since the intention of this predicate is to get the definition.